### DISCUSSION TITLE
``LensActionRequest::filters()`` returns empty or incomplete collection of applied filters.

### DISCUSSION THREAD

**Discussion by kusab85** [laravel/nova-issues#5702] [Ideas & Feature Requests]
- Laravel Version:  10.14.1
- Nova Version: v4.26.3 (Silver Surfer)
- PHP Version: PHP 8.2.4
- Database Driver & Version: SqlLite
- Operating System and Version: 10 22H2
- Browser type and version: #.#
- Reproduction Repository: https://github.com/kusab85/TestApp/tree/LensActionRequest.availableFilters

### Description:

When ``Lens`` has its own set of filters defined ``$request->filters()`` returns empty or incomplete collection of applied filters while handling ``LensActionRequest``.

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Clone repo locally.
2. Seed DB.
3. Navigate to lens: ``/nova/resources/users/lens/most-active-commentators``
4. ``git checkout current``
5. Tinker around with exporting filtered out resources
6. Check filenames of exported csv's! They all DO NOT HAVE applied filters names and values included.
7. ``git checkout expected``
8. Tinker around with exporting filtered out resources
9. Check filenames of exported csv's! As expected they all DO HAVE applied filters names and values included.
10. Analyze diffs: ``git diff current``

**Comment by crynobone** [comment]
Standalone shouldn't query the database so filters shouldn't be applied. Unable to qualify this as a bug report and moving this to feature request.
